.github/workflows/conformance-datapath.yaml

```diff
@@ -362,6 +360,12 @@ jobs:
           if [ "${{ matrix.ipv6 }}" != "false" ]; then
             IPV6="--helm-set=ipv6.enabled=true"
           fi
+          MASQ=""
+          if [ "${{ matrix.kpr }}" == "strict" ]; then
+            # BPF-masq requires KPR=strict.
+            # Disable IPv6 until https://github.com/cilium/cilium/issues/14350 has been resolved
+            MASQ="--helm-set=bpf.masquerade=true --helm-set=enableIPv6Masquerade=false"
+          fi
```

Comment:
> ...but enableIPv6Masquerade is still false,

If not specified, it defaults to `true` - https://github.com/cilium/cilium/blob/main/install/kubernetes/cilium/values.yaml#L1527

> ...but bpf.masquerade is now set to true only when KPR=strict

This is because of https://github.com/cilium/cilium/blob/main/daemon/cmd/daemon.go#L940 (KPR=strict means that NodePort is enabled).

Reply:
Very good question. I wanted to figure out myself. Anyway:
> level=warning msg="Falling back to iptables-based masquerading." error="BPF masquerade requires NodePort (--enable-node-port=\"true\")" subsys=daemon

To the agent automatically switched to the iptables-based masquerading.